code/modules/power/prototype_shield_emitter.dm

```diff
@@ -95,8 +95,8 @@
 		log_game("[src] turned off at [AREACOORD(Turf)]")
 
 /obj/machinery/power/proto_sh_emitter/interact(mob/user)
-	var/turf/T = /turf
-	var/turf/Turf = get_turf(src)
+	var/turf/ShieldedTurf = /turf
```

Comment:
Why is this assigned the turf type path?

Reply:
i used it to make sure to have block() check the turf, but as you said on the other review block() already does that, so is useless, i'll remove it